src/lib/config/searchConfig.js

```diff
@@ -432,19 +432,19 @@ const searchConfig = {
   acqVendors: {
     filters: [],
     sortBy: {
-      onEmptyQuery: 'name',
+      onEmptyQuery: 'bestmatch',
```

Comment:
thanks for fixing this, does the search works now in the vendors page?

Reply:
It doesn't crash now, but I don't see it having any effect or the results ordered alphabetically.